examples/protocol_specification_ex/http.yaml

```diff
@@ -19,3 +19,10 @@ speech_acts:
     headers: pt:str
     bodyy: pt:bytes
 ...
+---
+reply:
+  request: [response]
+  response: []
+roles: {client, server}
+end_states: []
```

Comment:
`closed` in line with above

Reply:
Again, I don't think close is the right name, either leave empty or name it something like
`end_states: [request_response_exchange]`.